src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -12,44 +12,47 @@
 import com.google.codeu.data.User;
 import org.jsoup.Jsoup;
 import org.jsoup.safety.Whitelist;
+import org.commonmark.node.*;
+import org.commonmark.parser.Parser;
+import org.commonmark.renderer.html.HtmlRenderer;
 
 /**  Handles fetching and saving user data. */
 
 @WebServlet("/about")
 public class AboutMeServlet extends HttpServlet {
-  
+
```

Comment:
please remove unnecessary whitespaces in this file. These add lines to the diff where there no changes, which is confusing (you can use an automatic formatter to do this) . Ditto below.